src/main/js/utils/Schedule.js

```diff
@@ -6,15 +6,17 @@ class Schedule extends Set {
    * The data parameter will be of type Schedule to copy or
    * type Integer to specify the gap time
    */
-  constructor(data = 15) {
+  constructor(data = 15, sort) {
```

Comment:
Again, we should avoid adding option parameters like this.

Reply:
Removed.